impl/src/test/groovy/com/okta/sdk/impl/resource/log/LogsTest.groovy

```diff
@@ -145,4 +151,38 @@ class LogsTest {
         assertThat log.getRequest().getIpChain().get(0).getGeographicalContext().getGeolocation().lat, equalTo(43.3091d)
         assertThat log.getRequest().getIpChain().get(0).getGeographicalContext().getGeolocation().lon, equalTo(-71.6861d)
     }
+
+    @Test
+    void testGetLogsBetweenDates() {
+
+        // get the events between the 'until' and 'since' date params
+        Date untilDate = Date.from(Instant.from(DateTimeFormatter.ISO_DATE_TIME.parse("2017-11-30T21:16:00.081Z")))
+        Date sinceDate = Date.from(Instant.from(DateTimeFormatter.ISO_DATE_TIME.parse("2017-11-30T21:15:16.838Z")))
+
+        List<LogEvent> logs = client.getLogs(untilDate, sinceDate, null, null, null).stream().collect(Collectors.toList())
```

Comment:
This method needs to verify the format of `untilDate` and/or `sinceDate` that would be sent over the wire, either with `MockClient.withMockResponse(...)` or `verify(mockClient.mockRequestExecutor).*`
(or something similar)

Reply:
Updated.